torch/testing/_internal/common_optimizers.py

```diff
@@ -1459,6 +1479,13 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
                 "TestOptimRenewed",
                 "test_forloop_goes_right_direction",
             ),
+            DecorateInfo(
+                unittest.skip(
+                    "RAdam does not support capturable single tensor https://github.com/pytorch/pytorch/issues/118230"
```

Comment:
I think this should be a skip in the test

Reply:
So in this case, I made it an actual skip. Since we are testing forloop directly, I felt like it made sense to fully skip here vs redundant tests with capturable=True and capturable=False

Reply:
With this skip, the noncapturable configs don't get run though, and that's worse for coverage.